\documentclass[../Bio_chemistryReview.tex]{subfiles}

\begin{document}
\chapter{Molecular Biology}

\section{DNA Structure} 

\subsection{General Overview}

\vocab{Purines} the precursor to G and A\\
\vocab{Pyrimidines} the precursor to T and C\\
A nucleoside is a ribose with a purine(pyrimidine) linked to the 1' carbon in a
$ \beta $-N-glycosidic linkage (N denotes that the bond is to a Nitrogen)
therefore the aromatic base is above the plane of the ribose. The nucleosides
are named as follows: 
\begin{multicols}{3}
  \setlength{\parindent}{0pt}
  \begin{center}
    A-ribose - adenosine\\
    G-ribose - guanosine\\
    C-ribose - cytosine\\
    T-ribose - thymidine\\
    U-ribose - uridine\\
  \end{center}
\end{multicols}

Both bases have abundant hydrogen bonding potential. \emph{Nucleotides} are
phosphate esters of nucleosides with up to three phosphate groups joined to the
ribose by the 5' hydroxy group. The \vocab{backbone} is the ribose + phosphate.
The bases are as follows:\newline 
\hfill \newline
PYRIMIDINE BASES
\begin{figure}[h]
  \centering
  \setatomsep{2em}
  \chemname{\chemfig{[:90]H-N*6(-(=[7]O)-N=(-[2]NH_{2}) -=-)}}{Cytosine} \qquad
  \chemname{\chemfig{[:90]H-N*6(-(=[7]O)-N(-[1]H)-(=[2]O)-(-[3]CH_{3})=-)}}{Thymine}
  \qquad \chemname{\chemfig{[:90]H-N*6(-(=[7]O)-N(-[1]H)-(=[2]O)-=-)}}{Uracil}
\end{figure}
\newline
PURINE BASES
\begin{figure}[h]
  \centering
  \setatomsep{2em}
  \chemname{\chemfig{[:85]H-N*5(-(*6(-N=-N=(-[2]NH_{2})-))=-N=-)}}{Adenine} \qquad
  \chemname{\chemfig{[:90]H-N*5(-(*6(-N=(-[7]NH_{2})-N(-[1]H)-(=[2]O)-))=-N=-)}}{Guanine}
\end{figure}
\newpage
The structure of dATP is below (Figure \ref{fig:dATP}) and gives the representative picture of a nucleotide.

\begin{figure}[h]
  \centering
  \includegraphics[scale=0.1]{dATP.jpg}
  \caption{Structure of dATP}
  \label{fig:dATP}
\end{figure}

\subsection{Polynucleotides}
Nucleotides covalently linked by \vocab{phosphodiester bonds} between the $
3^{\prime} $ hydroxy group of one deoxyribose and the $ 5^{\prime} $ phosphate
group of the next. Reaction causes the hydrolysis of the pyrophosphate molecule
driving the reaction forward (see Figure \ref{fig:Poly}):

\begin{figure}[H]
  \centering
  \includegraphics[scale=0.1]{PolyNucleotide.jpg}
  \caption{Polynucleotide Chain Backbone}
  \label{fig:Poly}
\end{figure}

\subsection{Watson-Crick Model of DNA Structure}
DNA is right-handed double helix held together by H bonds between bases (Cf.
next figure for H bonding). The strands are antiparallel (3' vs. 5') thus when
determining if two strands are complementary read one of them in reverse
direction with respect to the other. A,T (2 H bonds) and G,C (3 H bonds) see
Figure \ref{fig:bp}.
\begin{figure}[h]
  \centering
  \includegraphics[scale=0.2]{BasePair.jpg}
  \caption{Base Pairing}
  \label{fig:bp}
\end{figure}

Note: each pair contains one purine and one pyrimidine $ \implies $ can
determine number of purines from pyrimidines and vice versa. \vocab{Annealing
(hybridization)} is the binding of two complementary strands of DNA into double
strand (ds). \vocab{Denaturation} is their separation. The temperature at which
a solution of DNA is 50\% denatured is called $ \boldsymbol{ T_{m} } $.\\
The bases lie in plane $ \perp $ to length of DNA stacked 3.4 Ang. from each
other and experience hydrophobic interactions between each other also helping to
stabilize structure. Width is always 20 Ang., DNA completes a full turn every 10
base pairs i.e. 34 Ang. 

\subsection{Chromosome Structure and Packing}
Total genetic info = \vocab{genome}. Large linear piece = \vocab{chromosome}.
Prokaryotic genomes composed of single circular chromosome. Human genome
consists of over $ 10^{9} $ base pairs. Prokaryotes coil their circular
chromosome into \vocab{supercoils} using \vocab{DNA gyrase}.\\
Eukaryotic DNA wrapped around octamers of \vocab{histones} and are called
\vocab{nucleosomes}. The fully packed DNA is called \vocab{chromatin} composed
of closely stacked nucleosomes. The following Figure \ref{fig:sumDNA} summarizes
DNA:
\begin{figure}[h]
  \centering
  \includegraphics[scale=0.25]{DNAsum.jpg}
  \caption{Summary of DNA structure}
  \label{fig:sumDNA}
\end{figure}
Chromosomes can be stained with chemicals producing distinct light and dark
regions. Darker are more dense and called \vocab{heterochromatin} which is rich
in repeats. Light are called \vocab{euchromatin} and have a higher
transcription rate $ \implies $ higher gene activity.

\subsection{Centromeres}
Region of chromosome where spindle fibers attach (via \vocab{kinetochores})
during cell division. The centromere is made of heterochromatin and have p
(short) and q (long) arms.

\subsection{Telomeres}
Located at the end and consist of distinct, repeated nucleotide sequences usu.
6-8 bp long and guanine rich 5'-TTAGGG-3'. Can be single or double stranded DNA.
If single it can loop around to form a knot and stabilize the chromosome.
Telomeres \emph{prevent chromosome deterioration and fusion with neighboring
chromosomes.} Prokaryotes don't have telomeres due to circular genomes.

\section{Genome Structure and Genomic Variations}
Genome is 24 different chromosomes, 3.2 bn bp, and 21,000 genes. Has regions of
high transcription rates separated by \vocab{intergenic regions} composed of
noncoding DNA that may direct the assembly of specific chromatin structures,
regulation of nearby genes, or have no known purpose. A \vocab{gene} is a DNA
sequence that codes for a \vocab{gene product}. It includes both regulatory
regions (promoters and transcription stop sites) and a region for either a
protein or a non-coding RNA.

\subsection{Nucleotide Variation}
Predicted that there are single nucleotide changes once in every 1 kbp called
\vocab{single nucleotide polymorphisms (SNP)}. E.g. ability to taste PTC. Since
human genome is just over 3 billion bp then there are approximately 3 million
SNPs.

\subsection{Copy Number Variation}
Copy number variations are structural variations in the genome that lead to
different copied DNA sections. This can refer to large duplications of portions
of the genome or deletion. They are a normal part of the genome but have been
associated with cancer and other diseases e.g. Huntington's disease.

\subsection{Repeated Sequences: Tandem Repeats}
Short sequences of nucleotides repeated one right after the other. Repeats can
be unstable if repeating units are short or very long and may lead to chromosome
breaks and other diseases.

\subsection{Repeated Sequences: Transposons}
\vocab{Transposons} are mobile genetic elements i.e.\ jump around the genome and
can cause mutations and chromosome changes (inversions, deletions and
rearrangements). Three types of transposons: 
\begin{enumerate}
  \item IS - transposase gene flanked by inverted repeat sequences such as
    AACAATGG  --  CCATTGTT 
  \item Complex Transposon - Like IS but with several other genes in the flanked
    region along with the transposase
  \item Composite Transposon - Two similar or identical IS elements with a
    central region in between them 
\end{enumerate}

\vocab{Transposase} catalyzes the excision of transposon from donor site and
integration into a new genetic location. Can be excised and moved or duplicated
and moved. The inverted repeats are important for this mobilization.

\section{The Role of DNA}

\subsection{The Genetic Code}

\vocab{Transcription} - reading DNA and writing into RNA. \vocab{Translation}
- production of proteins from the mRNA via ribosomes. A nucleic acid is coded
for by a three letter sequence called a \vocab{codon}. E.g. the codon GTG in
DNA is transcribed into the RNA sequence CAC. There are 64 codons in total 3 of
which are \vocab{stop codons}. Two or more codons coding for same aa are called
\vocab{synonyms.}

\section{DNA Replication}

\vocab{Replication} occurs during the \emph{S phase}. DNA replication occurs in
three ways possible ways (bolded is actual): \emph{conservative, dispersive,
  \vocab{semi-conservative}}. DNA is uncoiled via \emph{helicase} and uses ATP
  to break the H bonds. \vocab{Origin of replication} (ORI) is the non-random
  location where helicase begins to unwind DNA. Generally, a protein complexes
  scans chromosome for ORI then calls in initiators of replication.\par
  \vocab{Topoisomerases} cut one or both of the strands and unwrap the helix.
  \vocab{Single-strand binding proteins (SSBPs)} protect and separate the ends
  of the single strands, which are referred to as \vocab{open complex}. DNA
  polymerase can only add nucleotides to existing chain. Thus \vocab{Primase}
  creates an RNA primer for each template strand. DNA polymerase adds dNTP's to
  3' end via displacement of the dNTP's 5' pyrophosphate. Hence, daughter strand
  is made 5' to 3' thus template strand reads 3' to 5'.  The point of unwinding
  at any instantaneous time is called the \vocab{replication fork}. To construct
  the other two strands (whose template strand has the $ 3^{\prime} $ end wound)
  a different method is needed (can't build $ 3^{\prime} $ to $ 5^{\prime} $).
  So, as DNA unwinds, primase lays down primer and DNA pol begins
  polymerization.  Process is repeated. Thus, they \emph{lag} behind the other
  two constructing strands (\vocab{leading strands}). Each component of lagging
  strand is called \vocab{Okazaki fragments}.

\begin{enumerate}
  \item \textbf{DNA replication is semiconservative}\\
    Each daughter genome contains one chain of parental DNA.
  \item \textbf{Polymerization occurs in the $ 5^{\prime} $ to $ 3^{\prime} $ direction}\\
    \emph{Never $ 3^{\prime} $ to $ 5^{\prime} $ addition} 
  \item \textbf{DNA pol requires a \emph{template}}
  \item \textbf{DNA pol requires a \emph{primer}}
  \item \textbf{Replication forks grow away from the origin in both directions}
  \item Replication of leading strand is \emph{continuous} while lagging strand
    is not 
  \item \textbf{All RNA primers are replaced by DNA} and \textbf{fragments are
    joined by DNA ligase}. RNA primers converted to DNA by DNA pol using a
    previous length of upstream DNA to replace the primer. Remember, the
    daughter strands run in opposite directions so this makes sense.
\end{enumerate}

\subsection{DNA Polymerase}

DNA pol is \emph{processive} (able to catalyze consecutive reactions w/o
releasing substrate). Prokaryotes have 5 different types of DNA pol. MUST KNOW
III and I: 
\begin{enumerate}
  \item \vocab{DNA pol III} - super-fast, super-accurate elongation of
    leading strand. Has $ 3^{\prime} \text{ to } 5^{\prime} $ exonuclease
    activity (can remove nucleotide at the end) so can go backwards and
    remove incorrectly added nucleotides; \vocab{proofreading function}. No   
  \item \vocab{DNA pol I} - add nucleotides at RNA primer called $ 5^{\prime}
    \text{ to } 3^{\prime} $ polymerase activity. Also, has exonuclease
    activity. Removes RNA primer via $ 5^{\prime} \text{ to } 3^{\prime} $
    exonuclease activity. Important for excision repair.  
  \item DNA pol II - has 5' to 3' polymerase activity and 3' to 5' exonuclease
    proofreading.  DNA repair and backup for DNA pol III 
  \item DNA pol IV and V - Similar characteristics. Stall other polymerase
    enzymes at replication forks during repair.
\end{enumerate}

\subsection{Eukaryotic vs. Prokaryotic Replication}

Eukaryotes have multiple ORI. Prokaryotes only have one. As chromosome is
duplicated it looks like $ \theta $ and so it is said to proceed via the
\vocab{theta mechanism} and is called \vocab{theta replication}.

\subsection{Replicating Telomeres}

The mechanism of the lagging strand presents an issue when replication fork
reaches the end of the chromosome. At the end a primer cannot be placed in
position so the end of the chromosome cannot be replicated. Thus each time the
chromosome is replicated it is shortened. The ends of the chromosomes contain
repeating DNA segments called \vocab{telomeres} which have no function but to
be buffers to protect the useful DNA from not being fully replicated at the end.
After telomeres become too short, cells can activate DNA repair pathways, enter a
senescent state (alive but will not divide), or enter apoptosis. The
\emph{Hayflick limit} is the number of times a normal human cell can divide
until telomere length stops cell division. \vocab{Telomerase} adds the
repetitive nucleotide sequences to the ends of chromosomes and therefore
lengthens telomeres. (Uncontrolled use of telomerase can lead to cancer).

\section{Genetic Mutation}
\emph{Germline mutations} - occur in germ cells $ \implies $ can pass to
offspring\\
\emph{Somatic mutations} - occur in somatic cells $ \implies $ aren't passed to
offspring, affects individual only.

\subsection{Causes of Mutations}

\subsubsection{Physical Mutagens}
Ionizing radiation can cause DNA breaks. One strand breaking is not bad, but if
both strands break near each other then hard to repair and may lead to
mutations. UV light may cause pyrimidines to become covalently linked if they
are next to each other leading to distortion of the DNA.

\subsubsection{Reactive Chemicals}
Chemicals can covalently alter bases or cause cross-linking (abnormal covalent
bonds between different parts of DNA) or strand breaks. Any compound that can
cause mutations is a \vocab{mutagen}. Compounds with similar structure as
purines and pyrimidines (large flat aromatic ring structures) can insert
themselves between base pairs (\emph{intercalcating}) and distort DNA structure.

\subsubsection{Biological Processes and Agents}
Incorrect basepair may not be repaired. Lysogenic viruses that insert into
genome can cause mutations and disrupt genetic function (some may even cause
cancer). Transposons as well.

\subsection{Types of Mutations}
Seven kinds:
\begin{enumerate}
  \item \vocab{Point mutations} - single pair substitutions. Can be
          \emph{transitions} (substitution of a pyrimidine(purine) for another
          pyrimidine(purine)) or \emph{transversion} (substitution of
          pyrimidine(purine) with a purine(pyrimidine). Three types:
          \begin{enumerate}
            \item \vocab{Missense mutation}: causes one aa to be replaced
              with another.  
            \item \vocab{Nonsense mutation}: stop codon
              replaces a regular codon 
            \item \vocab{Silent mutation}: codon
              is changed to another codon that codes for the same aa. (no
              harm) 
          \end{enumerate}
        \item \vocab{Insertions} - addition of one or more extra nucleotides
          into DNA sequence.  
        \item \vocab{Deletions} - removal of nucleotides
          from DNA sequence.
          \begin{itemize}
            \item \vocab{Frameshift mutation} - Caused by (2) and (3), when
              insertion or deletion of n nucleotides changes how DNA sequence is
              read and changes all of the aa coded for. Note: n$ \mod 3 \neq 0 $
              must hold 
          \end{itemize}
        \item \vocab{Inversions} - segment is reversed end to end.
        \item \vocab{Amplifications} - segment is duplicated
        \item \vocab{Translocations and rearrangements} - recombinations
          between nonhomologous chromosomes (cause of cancer).  
        \item \vocab{Loss of heterozygosity} - When one allele of a certain gene
          is lost due to deletion or a recombination event making the locus
          hemizygous.  Dangerous if the one remaining gene copy is defective.
\end{enumerate}

Transposons often cause mutations. They may insert in any part of the genome and
can affect gene expression or cause mutations (turn gene expression off, disrupt
protein-coding regions, disrupt a regulatory region and thus increase gene
expression). Can also cause structural changes when working in pairs. Two
cases:\\
\textbf{Case I:} (Transposons in the same direction)\\
\begin{figure}[H]
  \centering
  \includegraphics[scale=0.25]{SameDir.jpg}
  \caption{They line up parallel, the chromosomal segment between them is deleted during
recombination and takes one of the transposons with it. This deleted DNA +
transposon can then jump and insert into another segment of the chromosome
leading to rearrangement.}
\end{figure}

\noindent\textbf{Case II:} (Transposons inverted orientations)\\
\begin{figure}[H]
  \centering
  \includegraphics[scale=0.25]{InvertedDir.jpg}
  \caption{Pair and align with each other then after recombination, DNA segment between
them is inverted.}
\end{figure}

\subsection{Effects of Mutations}
Mutations in sex chromosomes have a greater effect than mutations in autosomes
since there are double copies of autosomes. Males only have one of X and Y and
females only express one of their X chromosomes so both are at risk since they
only effectively have one sex chromosome of each. Haploid expression in a
diploid organism is \vocab[]{hemizygosity}.
\begin{description}
  \item[\vocab{Gain-of-function mutation}] increase the activity of a certain
    gene product.
  \item[\vocab{Loss-of-function mutation}] decreases or completely suppresses
    activity of a certain gene product.
\end{description}
\vocab[]{Haploinsufficiency} is when a diploid organism has only a single
functional copy of a gene and this single copy is not enough for normal
function.

\subsection{Good and Bad Mutations}
Remember most mutations are neutral and evolution is based on mutations. Cancer
is driven by mutation accumulation. Cancer is driven by mutation accumulation.

\section{Types of DNA Repair}
Cell cycle checkpoints arrest cell cycle to check DNA. This occurs at
transition points such as the G\textsubscript{1}/S transition and the
G\textsubscript{2}/M transition. If DNA is too damaged apoptosis is induced.

\subsection{Direct Reversal}
Refers to DNA damage that can be fixed directly i.e. ``directly reversed''.
Photoreactivation is an example of this and occurs when enzymes repair
UV-induced pyrimidine photodimers using visible light. If left unrepaired this
leads to melanoma (skin cancer).

\subsection{Homology-Dependent Repair}
\vocab[]{Homology-dependent repair} relies on the undamaged, complementary DNA
strand to repair the damaged portions on the opposite strand. This can occur
before DNA replication (\vocab[]{excision repair}) or during and after
replication (post-replication repair).

\subsubsection{Excision Repair} 
Excision repair involves removing defective bases or nucleotides and replacing
them. These damaged bases can lead to mutations if replication machinery cannot
pair them properly.

\subsubsection{Post-Replication Repair}
The \vocab[]{mismatch repair pathway} (MMR) targets mismatched base pairs that
were not repaired by DNA polymerase proofreading during replication. To know
which base is the right one and which is wrong in a mismatched pair, some
bacteria use genome methylation on the older DNA strand. Other prokaryotes and
most eukaryotes rely on where the newly synthesized strand is recognized by the
free 3'-terminus on the leading strand, or by the presences of gaps between
Okazaki fragments on the lagging strand.

\subsection{Double-Strand Break Repair}
DNA double-strand breaks (DSB) can be caused by
\begin{enumerate}
  \item reactive oxygen species
  \item ionizing radiation
  \item UV light
  \item chemical agents
\end{enumerate}
Two repair options
\begin{itemize}
  \item \vocab{Homologous recombination}
  \item \vocab{Nonhomologous end-joining}
\end{itemize}
The goal is to reattach and fuse chromosomes that have come apart because of
DSB.

\subsubsection{Homologous Recombination}
After DNA replication, the genome contains identical sister chromatids.
Homologous recombination is where one sister chromatid can help repair a DSB in
the other (see Figure \ref{fig:HomoRecomb}). The steps are:
\begin{enumerate}
  \item DSB is identified and trimmed at 5' ends to generate single-stranded
    DNA. This is done by nucleases (which break phosphodiester bonds) and
    helicase (to unwind the DNA).
  \item Many proteins bind these ends and start a search of the genome to find a
    sister chromatid region that is complementary to the single-stranded DNA.
  \item Complementary sequences are used as a template to repair and connect the
    broken chromatid. This requires a ``joint molecule'' (see Figure
    \ref{fig:HomoRecomb}) where damaged and undamaged sister chromatids cross
    over.
  \item DNA polymerase and ligase build a corrected DNA strand.
\end{enumerate}

\begin{figure}[H]
  \centering
  \includegraphics[scale=0.2,frame]{HomologousRecombination.jpg}
  \caption{\textbf{Homologous Recombination to Repair Double-Strand Breaks}}
  \label{fig:HomoRecomb}
\end{figure}

\subsubsection{Nonhomologous End Joining}
Cells that aren't actively growing or cycling through the cell cycle don't have
a sister chromatid to repair a DSB in an error-free way. Nonhomologous end
joining is common in eukaryotes but relatively uncommon in prokaryotes. Process
proceeds as:
\begin{enumerate}
  \item Broken ends are stabilized and processed
  \item DNA ligase connects the fragments
\end{enumerate}
The goal is to just reconnect broken chromosomes and so often this can result in
base pairs being lost, or chromosomes being connected in an abnormal way.

\section{Gene Expression: Transcription}
\vocab[]{Gene expression} refers to the process whereby the information
contained in genes begins to have effects in the cell.

\subsection{Characteristics of RNA}
RNA is chemically distinct from DNA in three important ways:
\begin{enumerate}
  \item RNA is single-stranded, except in some viruses
  \item RNA contains uracil instead of thymine
  \item The pentose ring in RNA is ribose rather than $ 2^{\prime} $ deoxyribose
\end{enumerate}
RNA tends to be less stable because $ 2^{\prime} $ hydroxyl on ribose can
nucleophilically attack the backbone phosphate group. But this is not a big deal
because RNA is only needed for a short time. This is also why RNA contains U
instead of T because U, though less stable, costs less energy to produce.

\subsection{Types of RNA}
\subsubsection{Coding RNA}
\vocab[]{Messenger RNA} (mRNA) is the only type of coding RNA. mRNA have several
regions:
\begin{enumerate}
  \item $ 5^{\prime} $ untranslated region ($ 5^{\prime} $ UTR) for initiation
    and regulation
  \item \vocab{Open reading frame} (ORF) the region coding for the protein
  \item $ 3^{\prime} $ end not translated but often contains regulatory regions
    that influence post-transcriptional gene expression
\end{enumerate}
Eukaryotic mRNA is usually \vocab[]{monocistronic} $ \implies $ ``one gene, one
protein''. Prokaryotic mRNA is usually \vocab[]{polycistronic}. The first type
of RNA produced in the synthesizing mRNA for eukaryotes is
\vocab[]{heterogeneous nuclear RNA} (hnRNA). Adding cap and tail, etc. needed to
become mature mRNA.

\subsubsection{Non-Coding RNA}
These RNA are not translated protein and serve functional purposes. Two major
types for the MCAT:
\begin{enumerate}
  \item \vocab[]{tRNA} - transfer RNA carries amino acids from cytoplasm to the
    ribosome.
  \item \vocab[]{rRNA} - ribosomal RNA is the major component of the ribosome.
    Catalytic RNAs are also called \vocab[]{ribozymes} and are capable of
    performing specific biochemical reactions, similar to protein enzymes.
    Examples of other noncoding RNA on pg.\ 157.
\end{enumerate}

\subsection{Replication vs. Transcription}
Both replication and transcription involve \vocab[]{template-driven
polymerization} $ \implies $ RNA transcript is also complementary to the DNA template it
was derived from as in replication. Also, transcription goes from $ 5^{\prime} $
to $ 3^{\prime} $ direction and does not require a primer as in replication. It
also has no exonuclease activity i.e. it cannot fix errors it makes. The
sequence of nucleotides on the DNA that begins transcription is called the
\vocab[]{promoter} and the point where RNA polymerization actually starts is
called the \vocab[]{start site}.

\subsection{Reference Points in Transcription}
The mRNA is the complement of the transcribed DNA template (aka non-coding,
transcribed, or antisense strand) strand as expected. The other DNA strand is
called the coding or sense strand and has the same sequence as the transcript
(except with T's instead of U's). \vocab[]{Downstream} refers to the direction
the RNA pol moves towards and \vocab[]{upstream} is the opposite direction. See
Figure \ref{fig:refPoints}.
\begin{figure}[H]
  \centering
  \includegraphics[scale=0.3,frame]{RefTrans.jpg}
  \caption{Reference Points in Transcription}
  \label{fig:refPoints}
\end{figure}

\subsection{Prokaryotic Transcription}
In bacteria all RNA made by the same RNA polymerase consisting of a
\vocab[]{core enzyme} of five subunits ($ \alpha_{2}\beta\beta^{\prime}\omega $)
responsible for rapid elongation of the transcript Note: 2 alpha subunits. This
is incapable of initiating, for that you need the \vocab[]{sigma factor} to form
the \vocab[]{holoenzyme} which just means the complete enzyme. The sigma factor
helps in two ways:
\begin{enumerate}
  \item Greatly increase ability of RNA pol to recognize promoters
  \item Decrease the nonspecific affinity of holoenzyme for DNA
\end{enumerate}
After transcription begins, the sigma factor leaves.\par

Transcription occurs in three stages: \vocab[]{initiation},
\vocab[]{elongation}, and \vocab[]{termination}. Initiation occurs when the RNA
pol holoenzyme binds to a promoter usu.\ containing two primary sequences: the
\vocab[]{Pribnow box} at --10 and the \vocab[]{--35 sequence}. The complex is
called a \vocab[]{closed complex}. The RNA pol itself bound to the promoter is
called a \vocab[]{open complex} and once formed transcription can begin.\par

The core enzyme elongates the RNA chain \textit{processively} with one
polymerase complex synthesizing an entire RNA molecule. As it does so it moves
along the DNA downstream in a \vocab[]{transcription bubble} (see Figure
\ref{fig:refPoints}).

\subsection{Comparing Prokaryotic and Eukaryotic Transcription}
Four major differences between eukaryotic and prokaryotic transcription

\subsubsection{Location}
Prokaryotes transcribe and translate in cytoplasm and can do it simultaneously,
whereas eukaryotes transcribe in the nucleus and translate in the cytoplasm and
thus do not occur simultaneously. Also, mRNA is the primary transcript in
prokaryotes whereas it is hnRNA for eukaryotes.\par

\paragraph{Eukaryotic Primary Transcript Modification}
The most important modification is \vocab[]{splicing}, which refers to the
process of removing \vocab[]{introns}, regions of RNA that do not express any
proteins as opposed to \vocab[]{exons} which do express proteins, and joining
the exons. Splicing is done by the \vocab[]{spliceosome} through two reactions:
\begin{enumerate}
  \item Attaches one end of the intron to the conserved adenine forming a loop
  \item Joins the two exons
\end{enumerate}
There are alternative splicings that introduce mixing of which exons are
expressed and therefore lead to greater genetic variability.\par
The hnRNA must also have a $ 5^{\prime} $ cap and a $ 3^{\prime} $ poly-A tail.
The cap is essential for translation, and the cap and poly-A tail prevent
digestion of the mRNA by exonucleases in the cell.

\subsection{RNA Polymerase}
For eukaryotes there are several different RNA polymerases:
\begin{itemize}
  \item \textbf{RNA Polymerase I} transcribes most rRNA
  \item \textbf{RNA polymerase II} transcribes hnRNA
  \item \textbf{RNA Polymerase III} transcribes tRNA
\end{itemize}

\section{Gene Expression: Translation}

\subsection{Transfer RNA}
Each tRNA has a stem-and-loop structure (see Figure \ref{fig:tRNA}). The
\vocab[]{anticodon} reads the mRNA codon to be translated. The amino acids
attach to the \vocab[]{amino acid acceptor site} which is always the amino acid
combination CCA.

\begin{figure}[H] 
  \centering 
  \includegraphics[scale=.2,frame]{tRNA.jpg}
  \caption{Depiction of tRNA structure} 
  \label{fig:tRNA} 
\end{figure}

\subsection{The Wobble Hypothesis}
The \vocab[]{Wobble Hypothesis} states that the first two codon-anticodon pairs
obey normal base pairing rules but the third is more flexible e.g. G normally
pairs with C but it may also pair with U. This allows for less tRNAs than there
are possible mRNAs codon sequences. 

\subsection{Amino Acid Activation}
\vocab[]{Amino acid activation} is the process of hydrolyzing two high-energy
phosphate bonds to provide the energy to attach an amino acid to its tRNA
molecule. The process occurs in several steps (see Figure \ref{fig:AAact}):
\begin{enumerate}
  \item Amino acid is attached to ATP to form \textit{aminoacyl} AMP.
  \item The PP\textsubscript{i} (which is a 2 ATP equivalent) is hydrolyzed to 2
    orthophosphates releasing a lot of energy.
  \item tRNA loading is driven by switching the AMP with the amino acid.
\end{enumerate}
\begin{figure}[H]
  \centering
  \includegraphics[scale=0.2,frame]{AminoAcidAct.jpg}
  \caption{Amino Acid Activation process. Note: water as a reactant has been
  left out of all reactions.}
  \label{fig:AAact}
\end{figure}

\subsection{Aminoacyl-tRNA Synthetases}
\vocab[]{Aminoacyl-tRNA synthetase enzyme} is specific to each amino acid and
recognizes the three dimensional structure of both the amino acid and the tRNA.
Overall, it serves two functions:
\begin{itemize}
  \item Specific and accurate amino acid delivery
  \item Thermodynamic activation of the amino acid
\end{itemize}

\subsection{The Ribosome}
The ribosome is composed of a larger subunit and a smaller subunit. More detail
about the compositions of these subunits is given in the book on page 167-168.
There are three sites on a ribosome and tRNAs travel through in this order
(see Figure \ref{fig:rib}):
\begin{description}
  \item[A site] where each new tRNA delivers its amino acid
  \item[P site] where the growing polypeptide chain is located during
    translation
  \item[E site] (\textit{exit}-tRNA site) is where a now-empty tRNA sits prior
    to its release from the ribosome
\end{description}
\begin{figure}[H]
  \centering
  \includegraphics[scale=0.25,frame]{ribosome.jpg}
  \caption{The Ribosome}
  \label{fig:rib}
\end{figure}

\subsection{Prokaryotic Translation}
Recall that transcription and translation occur in the same compartment and
furthermore at the same time. So as mRNA is being made multiple ribosomes will
attach to begin translation at multiple sites. Recall that mRNA is polycistronic
so that there are multiple start sites.  For initiation, instead of a promoter
region as in transcription there is a pyrimidine rich ribosome binding site called
the \vocab[]{Shine-Dalgarno sequence} located at --10. Translation occurs in
three steps:
\paragraph{Initiation}
Initiation begins with smaller subunit binding with two initiation proteins IF1
and IF3 and as a complex binds the mRNA. Then the first aminoacyl-tRNA
(aka the initiator tRNA which is always a \textit{formyl}methionine) along with
IF2, which is also bound to one GTP, joins. Before elongation starts all
initiation factors dissociate.
\paragraph{Elongation}
is a three step cycle:
\begin{enumerate}
  \item Second aminoacyl-tRNA enters the A site and hydrogen bonds with the
    second codon.
  \item \vocab{Peptidyl transferase} of the large subunit catalyzes formation of
    peptide bond between first and second amino acids.
  \item \vocab[]{Translocation} is when the first tRNA moves to the E site and
    the second tRNA still attached to the polypeptide moves to the P site and
    the next codon moves into the A site. 
\end{enumerate}
\paragraph{Termination}
occurs when a stop codon appears in the A site, so a \vocab[]{release
factor} will enter the A site instead of a tRNA. Prokaryotes have three release
factor proteins: two recognize stop codons and one leads to the dissociation of
the other two after peptide release. Finally the ribosome separates into its
subunits.

\subsection{Eukaryotic Translation}
The first amino acid is always methionine. To initiate translation, instead of
the Shine-Dalgarno sequence there is the Kozak sequence located a few
nucleotides before the start codon. Translation begins with the formation of the
initiation complex searching for a start codon starting from the $ 5^{\prime} $
capped end. There is a discussion of the different factors involved on page 172
and regulation. Eukaryotes have two elongation factors: eEF-1 has two subunits,
one that helps with entry of an aminoacyl-tRNA into the A site and one that
catalyzes the release of GDP. eEF-2 is the translocase. Additional factors
needed for peptide bond formation.

\subsection{Cap-Independent Translation}
Not all eukaryotic translation begins at the $ 5^{\prime} $ cap, referred to as
cap-dependent translation. Cap-independent translation, translation occurring in
the middle of the mRNA, can occur if there is an internal ribosome entry site
IRES. This is useful for producing essential proteins for the cell in times of
stress.

\section{Controlling Gene Expression}
All cell types have the same genome but some express different parts more than
others. There are many mechanisms to regulate this expression. This regulation
is principally done during transcription. Gene expression can also be controlled
through \vocab[]{epigenetics} which focuses on changes in gene expression that
are not due to changes in DNA sequences, but are either heritable or have a long
term effect. The three most commonly studied are DNA methylation, chromatin
remodeling, and RNA interference.

\subsection{DNA Methylation and Chromatin Remodeling}
Prokaryotic and eukaryotic DNA can be covalently modified by adding a methyl
group which plays an important role in controlling gene expression. It
turns off gene expression in eukaryotes in two ways:
\begin{enumerate}
  \item Methylation physically blocks the gene from transcriptional proteins
  \item Certain proteins bind methylated CpG groups and recruit chromatin
    remodeling proteins that change the winding of DNA around histones.
\end{enumerate}

\subsection{Gene Dose}
One way to increase gene expression is to increase the copy number of a gene by
amplification. Gene deletion causes a decrease in gene expression.

\subsection{Variations on Diploid Gene Expression}
In most cases both copies of a gene are either expressed or not. There are some
exceptions though.

\subsubsection{Imprinting}
\vocab[]{Genomic imprinting} is when only one allele of a gene is expressed.
Imprinted genes tend to be clustered together on chromosomes. Imprinting can
change from generation to generation supporting the observation that it is an
epigenetic process. Silencing of certain genes involves DNA methylation, histone
modification, and binding of long ncRNAs established in the germline and
maintained throughout life and mitotic divisions.

\subsubsection{X Chromosome Inactivation}
Females will inactivate one of their X chromosomes irreversibly which is
referred to as Xi while the active one is referred to as Xa. This inactivation
occurs during development, at the blastocyst stage, where each cell in the inner
cell mass randomly choses which X to inactivate. Thus, within the female body
there are lines of cells that have one of the X chromosomes active and the other
silenced while in a different line it may be the other way around. Xi is heavily
methylated.

\subsubsection{Regulation of Transcription: Prokaryotes}
Regulation of gene expression in prokaryotes is primarily done through
regulation of transcription. This is usually done by having promoters that are
stronger than others. Though this is a ``pre-set'' in the sense that it is not
flexible to changing conditions. Cellular processes are often anabolic
(building up) e.g. the trp operon, or catabolic (breaking down) e.g. the lac
operon. As a result, we expect the feedback inhibition to be such that too much
of a product will inhibit anabolic enzymes and too little of a product will
induce catabolic enzymes. \vocab[]{Operons} include two components, a coding
sequence for enzymes and upstream regulatory sequences or control sites.

\subsubsection{The Lac Operon} contains several components:
\begin{description}
  \item[P region:] the promoter site on DNA to which RNA polymerase binds to
    initiate transcription of Y, Z, and A genes.
  \item[O region:] the operator site to which the Lac repressor binds.
  \item[Z gene:] codes for the enzyme $ \beta $-galactosidase, which cleaves
    lactose into glucose and galactose.
  \item[Y gene:] codes for permease, a protein which transports lactose into the
    cell.
  \item[A gene:] codes for transacetylase, an enzyme which transfers an acetyl
    group from acetyl-CoA to $ \beta $-galactosidase (Note: not required for
    lactose metabolism).
\end{description}
Two regulatory genes at a distant site that code for proteins important in the
regulation of the lac operon's Z, Y, and A:
\begin{description}
  \item[crp gene:] codes for catabolite activator protein (CAP) coupling the lac
    operon to glucose levels in the cell
  \item[I gene:] codes for lac repressor protein
\end{description}

The repressor protein from the I gene can bind to the operator of the lac operon
and thus prevent RNA pol from binding the promoter. But if the repressor protein
binds to lactose, it cannot bind to the operator allowing for RNA pol to do its
job (see Figure \ref{fig:lac} for schematic).\par
\begin{figure}[H]
  \centering
  \includegraphics[scale=0.25,frame]{Lac.jpg}
  \caption{The Lac Operon in the Presence of Glucose and Absence of Lactose}
  \label{fig:lac}
\end{figure}
Glucose levels control adenlyl cyclase (low glucose activates it and high
glucose inactivates it), which converts ATP to cAMP required by the CAP protein
to bind to the promoter and assist binding of RNA pol. Thus, high glucose \imp
Lac is suppressed. Work out the regulation as an exercise.

\subsubsection{The Trp Operon}
Trp Operon allows for the production of tryptophan. The repressor protein is
coded by the trpR gene and requires binding to tryptophan to bind to the
operator and prevent transcription (See Figure \ref{fig:trp}).
\begin{figure}[H]
  \centering
  \includegraphics[scale=0.25,frame]{trp.jpg}
  \caption{The Trp Operon in the Presence of Tryptophan}
  \label{fig:trp}
\end{figure}

\subsection{Regulation of Transcription: Eukaryotes}
Most of the regulation of transcription in eukaryotes occurs at initiation. For
protein-coding genes, there are upstream control elements (UCEs) usually about
200 bases upstream of the initiation site.\par

Enhancer sequences which increase the likelihood that a particular gene will be
transcribed, are bound by activator proteins. This is another kind of
transcriptional regulation. These enhancer sequences may be many thousands of
base pairs away and thus require DNA looping to bring the enhancer and
transcribing region together.\par

Eukaryotes also have \vocab[]{gene repressor proteins}, which inhibit
transcription. \vocab[]{Transcription factors} have DNA-binding domains and are
crucial in transcription regulation. They can bind promoters or other regulatory
sequences. Often times regulation occurs through a large number of proteins
leading to a large network of proteins that can regulate functions across vast
distances of the network (think extracellular signaling). The binding of
transcriptional machinery to DNA is often regulated by extracellular
signals.\par

Other methods of transcriptional regulation include:
\begin{itemize}
  \item \vocab{RNA Translocation:} mRNA transcripts must be exported from the
    nucleus to the cytoplasm or to different areas of the cell. During this
    exportation they are transcriptionally silent. It is important for
    transcripts to be translated only when they get to the correct area of the
    cell.
  \item \vocab{mRNA Surveillance:} Cells check transcripts and if it is
    defective then it is degraded.
  \item \vocab{RNA Interference:} is a way to silence gene expression after a
    transcript has been made. It is mediated by miRNA and siRNA (talked about in
    section 5.7 in the book) where siRNAs bind complementary sequences on mRNAs
    and the ds-RNA is then degraded.
\end{itemize}

\paragraph{Post-Translational Modification}
These modifications include protein folding, additional processing, and
transportation to correct cell location.

\paragraph{Protein Folding}
Protein folding is aided by \vocab[]{chaperons}. Proteins folded correctly are
said to be in their native conformation. 

\paragraph{Covalent Modification}
Many proteins are covalently modified e.g. adding a hydrophobic group to
facilitate membrane localization. Also, methylation which is commonly done to
lysine or arginine. In addition to having things attached, proteins may also be
linked to other proteins.

\paragraph{Processing}
Many proteins require cleavage to become functional. This allows cells to
produce a lot of pre-functional proteins and then cleave them rapidly to produce
many functional proteins on short notice. Enzyme precursors are called
\vocab[]{zymogens} or \vocab[]{proenzymes}.

\section*{Beyond Nuclear Molecular Biology: Organelle Genomes}
There is discussion about mitochondria and the endosymbiotic theory of the cell.
I don't think that it is particularly relevant for the MCAT, but page 183 for
reference.

\section{Summary}
Transcription begins at a start site, but needs a promoter upstream of this. It
ends at a termination signal. The RNA transcript contains the open reading frame
(which goes from start codon to stop codon), as well as both $ 5^{\prime} $ and
$ 3^{\prime} $ regulatory regions (see Figure \ref{fig:GeneSummary} ).
\begin{figure}[H]
  \centering
  \includegraphics[scale=0.1,frame]{summary.jpg}
  \caption{Gene Structure and Protein Expression}
  \label{fig:GeneSummary}
\end{figure}
And a table (see Figure \ref{fig:RevTable}).
\begin{figure}[H]
  \centering
  \includegraphics[scale=0.3,frame]{SumTable.jpg}
  \caption{A Review of Molecular Biology Processes}
  \label{fig:RevTable}
\end{figure}


\end{document}
