\documentclass[../Bio_chemistryReview.tex]{subfiles}

\begin{document}
\chapter{Molecular Biology}

\section{DNA Structure} 

\subsection{General Overview}

\vocab{Purines} the precursor to G and A\\
\vocab{Pyrimidines} the precursor to T and C\\
A nucleoside is a ribose with a purine(pyrimidine) linked to the 1' carbon in a
$ \beta $-N-glycosidic linkage (N denotes that the bond is to a Nitrogen)
therefore the aromatic base is above the plane of the ribose. The nucleosides
are named as follows: 
\begin{multicols}{3}
  \setlength{\parindent}{0pt}
  \begin{center}
    A-ribose - adenosine\\
    G-ribose - guanosine\\
    C-ribose - cytosine\\
    T-ribose - thymidine\\
    U-ribose - uridine\\
  \end{center}
\end{multicols}

Both bases have abundant hydrogen bonding potential. \emph{Nucleotides} are
phosphate esters of nucleosides with up to three phosphate groups joined to the
ribose by the 5' hydroxy group. The \vocab{backbone} is the ribose + phosphate.
The bases are as follows:\newline 
\hfill \newline
PYRIMIDINE BASES
\begin{figure}[h]
  \centering
  \setatomsep{2em}
  \chemname{\chemfig{[:90]H-N*6(-(=[7]O)-N=(-[2]NH_{2}) -=-)}}{Cytosine} \qquad
  \chemname{\chemfig{[:90]H-N*6(-(=[7]O)-N(-[1]H)-(=[2]O)-(-[3]CH_{3})=-)}}{Thymine}
  \qquad \chemname{\chemfig{[:90]H-N*6(-(=[7]O)-N(-[1]H)-(=[2]O)-=-)}}{Uracil}
\end{figure}
\newline
PURINE BASES
\begin{figure}[h]
  \centering
  \setatomsep{2em}
  \chemname{\chemfig{[:85]H-N*5(-(*6(-N=-N=(-[2]NH_{2})-))=-N=-)}}{Adenine} \qquad
  \chemname{\chemfig{[:90]H-N*5(-(*6(-N=(-[7]NH_{2})-N(-[1]H)-(=[2]O)-))=-N=-)}}{Guanine}
\end{figure}
\newpage
The structure of dATP is below and gives the representative picture of a nucleotide.

\begin{figure}[h]
  \centering
  \includegraphics[scale=0.1]{dATP.jpg}
\end{figure}
\subsection{Polynucleotides}

Nucleotides covalently linked by \vocab{phosphodiester bonds} between the $
3^{\prime} $ hydroxy group of one deoxyribose and the $ 5^{\prime} $ phosphate
group of the next. Reaction causes the hydrolysis of the pyrophosphate molecule
driving the reaction forward (Cf below):

\begin{figure}[H]
  \centering
  \includegraphics[scale=0.1]{PolyNucleotide.jpg}
\end{figure}

\subsection{Watson-Crick Model of DNA Structure}

DNA is right-handed double helix held together by H bonds between bases (Cf.
next figure for H bonding). The strands are antiparallel (3' vs. 5') thus when
determining if two strands are complementary read one of them in reverse
direction with respect to the other. A,T (2 H bonds) and G,C (3 H bonds).
\begin{figure}[h]
  \centering
  \includegraphics[scale=0.2]{BasePair.jpg}
\end{figure}

Note: each pair contains one purine and one pyrimidine $ \implies $ can
determine number of purines from pyrimidines and vice versa. \vocab{Annealing
(hybridization)} is the binding of two complementary strands of DNA into double
strand (ds). \vocab{Denaturation} is their separation. The temperature at which
a solution of DNA is 50\% denatured is called $ \boldsymbol{ T_{m} } $.\\
The bases lie in plane $ \perp $ to length of DNA stacked 3.4 Ang. from each
other and experience hydrophobic interactions between each other also helping to
stabilize structure. Width is always 20 Ang., DNA completes a full turn every 10
base pairs i.e. 34 Ang. 

\subsection{Chromosome Structure and Packing}

Total genetic info = \vocab{genome}. Large linear piece = \vocab{chromosome}.
Prokaryotic genomes composed of single circular chromosome. Human genome
consists of over $ 10^{9} $ base pairs. Prokaryotes coil their circular
chromosome into \vocab{supercoils} using \vocab{DNA gyrase}.\\
Eukaryotic DNA wrapped around octamers of \vocab{histones} and are called
\vocab{nucleosomes}. The fully packed DNA is called \vocab{chromatin} composed
of closely stacked nucleosomes. The following summarizes DNA:

\begin{figure}[h]
  \centering
  \includegraphics[scale=0.25]{DNAsum.jpg}
\end{figure}

Chromosomes can be stained with chemicals producing distinct light and dark
regions. Darker are more dense and called \vocab{heterochromatin} which is rich
in repeats. Light are called \vocab{euchromatin} and have a higher
transcription rate $ \implies $ higher gene activity.

\subsection{Centromeres}

Region of chromosome where spindle fibers attach (via \vocab{kinetochores})
during cell division. The centromere is made of heterochromatin and have p
(short) and q (long) arms.

\subsection{Telomeres}

Located at the end and consist of distinct, repeated nucleotide sequences usu.
6-8 bp long and guanine rich 5'-TTAGGG-3'. Can be single or double stranded DNA.
If single it can loop around to form a knot and stabilize the chromosome.
Telomeres \emph{prevent chromosome deterioration and fusion with neighboring
chromosomes.} Prokaryotes don't have telomeres due to circular genomes.

\section{Genome Structure and Genomic Variations}

Genome is 24 different chromosomes, 3.2 bn bp, and 21,000 genes. Has regions of
high transcription rates separated by \vocab{intergenic regions} composed of
noncoding DNA that may direct the assembly of specific chromatin structures,
regulation of nearby genes, or have no known purpose. A \vocab{gene} is a DNA
sequence that codes for a \vocab{gene product}. It includes both regulatory
regions (promoters and transcription stop sites) and a region for either a
protein or a non-coding RNA.

\subsection{Nucleotide Variation}

Predicted that there are single nucleotide changes once in every 1 kbp called
\vocab{single nucleotide polymorphisms (SNP)}. E.g. ability to taste PTC. Since
human genome is just over 3 billion bp then there are approximately 3 million
SNPs.

\subsection{Copy Number Variation}

Copy number variations are structural variations in the genome that lead to
different copied DNA sections. They are a normal part of the genome but have
been associated with cancer and other diseases e.g. Huntington's disease.

\subsection{Repeated Sequences: Tandem Repeats}

Short sequences of nucleotides repeated one right after the other. Repeats can
be unstable if repeating units are short or very long and may lead to chromosome
breaks and other diseases.

\subsection{Repeated Sequences: Transposons}

\vocab{Transposons} are mobile genetic elements i.e.\ jump around the genome and
can cause mutations and chromosome changes (inversions, deletions and
rearrangements). Three types of transposons: 
\begin{enumerate}
  \item IS - transposase gene flanked by inverted repeat sequences such as
    AACAATGG  --  CCATTGTT 
  \item Complex Transposon - Like IS but with several other genes in the flanked
    region along with the transposase
  \item Composite Transposon - Two similar or identical IS elements with a
    central region in between them 
\end{enumerate}

\vocab{Transposase} catalyzes the excision of transposon from donor site and
integration into a new genetic location. Can be excised and moved or duplicated
and moved. The inverted repeats are important for this mobilization.

\section{The Role of DNA}

\subsection{The Genetic Code}

\vocab{Transcription} - reading DNA and writing into RNA. \vocab{Translation}
- production of proteins from the mRNA via ribosomes. A nucleic acid is coded
for by a three letter sequence called a \vocab{codon}. E.g. the codon GTG in
DNA is transcribed into the RNA sequence CAC. There are 64 codons in total 3 of
which are \vocab{stop codons}. Two or more codons coding for same aa are called
\vocab{synonyms.}

\section{DNA Replication}

\vocab{Replication} occurs during the \emph{S phase}. DNA replication occurs in
three ways possible ways (bolded is actual): \emph{conservative, dispersive,
  \vocab{semi-conservative}}. DNA is uncoiled via \emph{helicase} and uses ATP
  to break the H bonds. \vocab{Origin of replication} (ORI) is the non-random
  location where helicase begins to unwind DNA. Generally, a protein complexes
  scans chromosome for ORI then calls in initiators of replication.\par
  \vocab{Topoisomerases} cut one or both of the strands and unwrap the helix.
  \vocab{Single-strand binding proteins (SSBPs)} protect and separate the ends
  of the single strands, which are referred to as \vocab{open complex}. DNA
  polymerase can only add nucleotides to existing chain. Thus \vocab{Primase}
  creates an RNA primer for each template strand. DNA polymerase adds dNTP's to
  3' end via displacement of the dNTP's 5' pyrophosphate. Hence, daughter strand
  is made 5' to 3' thus template strand reads 3' to 5'.  The point of unwinding
  at any instantaneous time is called the \vocab{replication fork}. To construct
  the other two strands (whose template strand has the $ 3^{\prime} $ end wound)
  a different method is needed (can't build $ 3^{\prime} $ to $ 5^{\prime} $).
  So, as DNA unwinds, primase lays down primer and DNA pol begins
  polymerization.  Process is repeated. Thus, they \emph{lag} behind the other
  two constructing strands (\vocab{leading strands}). Each component of lagging
  strand is called \vocab{Okazaki fragments}.

\begin{enumerate}
  \item \textbf{DNA replication is semiconservative}\\
    Each daughter genome contains one chain of parental DNA.
  \item \textbf{Polymerization occurs in the $ 5^{\prime} $ to $ 3^{\prime} $ direction}\\
    \emph{Never $ 3^{\prime} $ to $ 5^{\prime} $ addition} 
  \item \textbf{DNA pol requires a \emph{template}}
  \item \textbf{DNA pol requires a \emph{primer}}
  \item \textbf{Replication forks grow away from the origin in both directions}
  \item Replication of leading strand is \emph{continuous} while lagging strand
    is not 
  \item \textbf{All RNA primers are replaced by DNA} and \textbf{fragments are
    joined by DNA ligase}. RNA primers converted to DNA by DNA pol using a
    previous length of upstream DNA to replace the primer. Remember, the
    daughter strands run in opposite directions so this makes sense.
\end{enumerate}

\subsection{DNA Polymerase}

DNA pol is \emph{processive} (able to catalyze consecutive reactions w/o
releasing substrate). Prokaryotes have 5 different types of DNA pol. MUST KNOW
III and I: 
\begin{enumerate}
  \item \vocab{DNA pol III} - super-fast, super-accurate elongation of
    leading strand. Has $ 3^{\prime} \text{ to } 5^{\prime} $ exonuclease
    activity (can remove nucleotide at the end) so can go backwards and
    remove incorrectly added nucleotides; \vocab{proofreading function}. No   
  \item \vocab{DNA pol I} - add nucleotides at RNA primer called $ 5^{\prime}
    \text{ to } 3^{\prime} $ polymerase activity. Also, has exonuclease
    activity. Removes RNA primer via $ 5^{\prime} \text{ to } 3^{\prime} $
    exonuclease activity. Important for excision repair.  
  \item DNA pol II - has 5' to 3' polymerase activity and 3' to 5' exonuclease
    proofreading.  DNA repair and backup for DNA pol III 
  \item DNA pol IV and V - Similar characteristics. Stall other polymerase
    enzymes at replication forks during repair.
\end{enumerate}

\subsection{Eukaryotic vs. Prokaryotic Replication}

Eukaryotes have multiple ORI. Prokaryotes only have one. As chromosome is
duplicated it looks like $ \theta $ and so it is said to proceed via the
\vocab{theta mechanism} and is called \vocab{theta replication}.

\subsection{Replicating Telomeres}

The mechanism of the lagging strand presents an issue when replication fork
reaches the end of the chromosome. At the end a primer cannot be placed in
position so the end of the chromosome cannot be replicated. Thus each time the
chromosome is replicated it is shortened. The ends of the chromosomes contain
repeating DNA segments called \vocab{telomeres} which have no function but to
be buffers to protect the useful DNA from not being fully replicated at the end.
After telomeres become too short, cells can activate DNA repair pathways, enter a
senescent state (alive but will not divide), or enter apoptosis. The
\emph{Hayflick limit} is the number of times a normal human cell can divide
until telomere length stops cell division. \vocab{Telomerase} adds the
repetitive nucleotide sequences to the ends of chromosomes and therefore
lengthens telomeres. (Uncontrolled use of telomerase can lead to cancer).

\section{Genetic Mutation}
\emph{Germline mutations} - occur in germ cells $ \implies $ can pass to
offspring\\
\emph{Somatic mutations} - occur in somatic cells $ \implies $ aren't passed to
offspring, affects individual only.

\subsection{Causes of Mutations}

\subsubsection{Physical Mutagens}
Ionizing radiation can cause DNA breaks. One strand breaking is not bad, but if
both strands break near each other then hard to repair and may lead to
mutations. UV light may cause pyrimidines to become covalently linked if they
are next to each other leading to distortion of the DNA.

\subsubsection{Reactive Chemicals}
Chemicals can covalently alter bases or cause cross-linking (abnormal covalent
bonds between different parts of DNA) or strand breaks. Any compound that can
cause mutations is a \vocab{mutagen}. Compounds with similar structure as
purines and pyrimidines (large flat aromatic ring structures) can insert
themselves between base pairs (\emph{intercalcating}) and distort DNA structure.

\subsubsection{Biological Processes and Agents}
Incorrect basepair may not be repaired. Lysogenic viruses that insert into
genome can cause mutations and disrupt genetic function (some may even cause
cancer). Transposons as well.

\subsection{Types of Mutations}
Seven kinds:
\begin{enumerate}
  \item \vocab{Point mutations} - single pair substitutions. Can be
          \emph{transitions} (substitution of a pyrimidine(purine) for another
          pyrimidine(purine)) or \emph{transversion} (substitution of
          pyrimidine(purine) with a purine(pyrimidine). Three types:
          \begin{enumerate}
            \item \vocab{Missense mutation}: causes one aa to be replaced
              with another.  
            \item \vocab{Nonsense mutation}: stop codon
              replaces a regular codon 
            \item \vocab{Silent mutation}: codon
              is changed to another codon that codes for the same aa. (no
              harm) 
          \end{enumerate}
        \item \vocab{Insertions} - addition of one or more extra nucleotides
          into DNA sequence.  
        \item \vocab{Deletions} - removal of nucleotides
          from DNA sequence.
          \begin{itemize}
            \item \vocab{Frameshift mutation} - Caused by (2) and (3), when
              insertion or deletion of n nucleotides changes how DNA sequence is
              read and changes all of the aa coded for. Note: n$ \mod 3 \neq 0 $
              must hold 
          \end{itemize}
        \item \vocab{Inversions} - segment is reversed end to end.
        \item \vocab{Amplifications} - segment is duplicated
        \item \vocab{Translocations and rearrangements} - recombinations
          between nonhomologous chromosomes (cause of cancer).  
        \item \vocab{Loss of heterozygosity} - When one allele of a certain gene
          is lost due to deletion or a recombination event making the locus
          hemizygous.  Dangerous if the one remaining gene copy is defective.
\end{enumerate}

Transposons often cause mutations. They may insert in any part of the genome and
can affect gene expression or cause mutations (turn gene expression off, disrupt
protein-coding regions, disrupt a regulatory region and thus increase gene
expression). Can also cause structural changes when working in pairs. Two
cases:\\
\textbf{Case I:} (Transposons in the same direction)\\
\begin{figure}[H]
  \centering
  \includegraphics[scale=0.25]{SameDir.jpg}
  \caption{They line up parallel, the chromosomal segment between them is deleted during
recombination and takes one of the transposons with it. This deleted DNA +
transposon can then jump and insert into another segment of the chromosome
leading to rearrangement.}
\end{figure}

\newpage
\noindent\textbf{Case II:} (Transposons inverted orientations)\\
\begin{figure}[H]
  \centering
  \includegraphics[scale=0.25]{InvertedDir.jpg}
  \caption{Pair and align with each other then after recombination, DNA segment between
them is inverted.}
\end{figure}

\subsection{Effects of Mutations}
Mutations in sex chromosomes have a greater effect than mutations in autosomes
since there are double copies of autosomes. Males only have one of X and Y and
females only express one of their X chromosomes so both are at risk since they
only effectively have one sex chromosome of each. Haploid expression in a
diploid organism is \vocab[]{hemizygosity}.
\begin{description}
  \item[\vocab{Gain-of-function mutation}] increase the activity of a certain
    gene product.
  \item[\vocab{Loss-of-function mutation}] decreases or completely suppresses
    activity of a certain gene product.
\end{description}
\vocab[]{Haploinsufficiency} is when a diploid organism has only a single
functional copy of a gene and this single copy is not enough for normal
function.

\subsection{Good and Bad Mutations}
Remember most mutations are neutral and evolution is based on mutations. Cancer
is driven by mutation accumulation. Cancer is driven by mutation accumulation.

\section{Types of DNA Repair}
Cell cycle checkpoints arrest cell cycle to check DNA. This occurs at
transition points such as the G\textsubscript{1}/S transition and the
G\textsubscript{2}/M transition. If DNA is too damaged apoptosis is induced.

\subsection{Direct Reversal}
Refers to DNA damage that can be fixed directly i.e. ``directly reversed''.
Photoreactivation is an example of this and occurs when enzymes repair
UV-induced pyrimidine photodimers using visible light. If left unrepaired this
leads to melanoma (skin cancer).

\subsection{Homology-Dependent Repair}
\vocab[]{Homology-dependent repair} relies on the undamaged, complementary DNA
strand to repair the damaged portions on the opposite strand. This can occur
before DNA replication (\vocab[]{excision repair}) or during and after
replication (post-replication repair).

\subsubsection{Excision Repair} 
Excision repair involves removing defective bases or nucleotides and replacing
them. These damaged bases can lead to mutations if replication machinery cannot
pair them properly.

\subsubsection{Post-Replication Repair}
The \vocab[]{mismatch repair pathway} (MMR) targets mismatched base pairs that
were not repaired by DNA polymerase proofreading during replication. To know
which base is the right one and which is wrong in a mismatched pair, some
bacteria use genome methylation on the older DNA strand. Other prokaryotes and
most eukaryotes rely on where the newly synthesized strand is recognized by the
free 3'-terminus on the leading strand, or by the presences of gaps between
Okazaki fragments on the lagging strand.

\subsection{Double-Strand Break Repair}
DNA double-strand breaks (DSB) can be caused by
\begin{enumerate}
  \item reactive oxygen species
  \item ionizing radiation
  \item UV light
  \item chemical agents
\end{enumerate}
Two repair options
\begin{itemize}
  \item \vocab{Homologous recombination}
  \item \vocab{Nonhomologous end-joining}
\end{itemize}
The goal is to reattach and fuse chromosomes that have come apart because of
DSB.

\subsubsection{Homologous Recombination}
After DNA replication, the genome contains identical sister chromatids.
Homologous recombination is where one sister chromatid can help repair a DSB in
the other (see Figure \ref{fig:HomoRecomb}). The steps are:
\begin{enumerate}
  \item DSB is identified and trimmed at 5' ends to generate single-stranded
    DNA. This is done by nucleases (which break phosphodiester bonds) and
    helicase (to unwind the DNA).
  \item Many proteins bind these ends and start a search of the genome to find a
    sister chromatid region that is complementary to the single-stranded DNA.
  \item Complementary sequences are used as a template to repair and connect the
    broken chromatid. This requires a ``joint molecule'' (see Figure
    \ref{fig:HomoRecomb}) where damaged and undamaged sister chromatids cross
    over.
  \item DNA polymerase and ligase build a corrected DNA strand.
\end{enumerate}

\begin{figure}[H]
  \centering
  \includegraphics[scale=0.2,frame]{HomologousRecombination.jpg}
  \caption{\textbf{Homologous Recombination to Repair Double-Strand Breaks}}
  \label{fig:HomoRecomb}
\end{figure}

\subsubsection{Nonhomologous End Joining}
Cells that aren't actively growing or cycling through the cell cycle don't have
a sister chromatid to repair a DSB in an error-free way. Nonhomologous end
joining is common in eukaryotes but relatively uncommon in prokaryotes. Process
proceeds as:
\begin{enumerate}
  \item Broken ends are stabilized and processed
  \item DNA ligase connects the fragments
\end{enumerate}
The goal is to just reconnect broken chromosomes and so often this can result in
base pairs being lost, or chromosomes being connected in an abnormal way.

\section{Gene Expression: Transcription}
\vocab[]{Gene expression} refers to the rpocess whereby te information contained
in genes begins to have effects in the cell.

\subsection{Characteristics of RNA}
RNA is chemically distinct from DNA in three important ways:
\begin{enumerate}
  \item RNA is single-stranded, except in some viruses
  \item RNA contains uracil instead of thymine
  \item The pentose ring in RNA is ribose rather than $ 2^{\prime} $ deoxyribose
\end{enumerate}
RNA tends to be less stable because $ 2^{\prime} $ hydroxyl can nucleophilically
attack the backbone phosphate group. But this is not a big deal because RNA is
only needed for a short time. This is also why RNA contains U instead of T
because U, though less stable, costs less energy to produce.

\subsection{Types of RNA}
\subsubsection{Coding RNA}
\vocab[]{Messenger RNA} (mRNA) is the only type of coding RNA. mRNA have several
regions:
\begin{enumerate}
  \item $ 5^{\prime} $ untranslated region ($ 5^{\prime} $ UTR) for initiation
    and regulation
  \item \vocab{Open reading frame} (ORF) the region coding for the protein
  \item $ 3^{\prime} $ end not translated but often contains regulatory regions
    that influence post-transcriptional gene expression
\end{enumerate}
Eukaryotic mRNA is usually \vocab[]{monocistronic} $ \implies $ ``one gene, one
protein''. Prokaryotic mRNA is usually \vocab[]{polycistronic}. The first type
of RNA produced in the synthesizing mRNA is \vocab[]{heterogeneous nuclear RNA}
(hnRNA). Adding cap and tail, etc. needed to become mature mRNA.

\subsubsection{Non-Coding RNA}
These RNA are not translated protein and serve functional purposes. Two major
types for the MCAT:
\begin{enumerate}
  \item \vocab[]{tRNA} - transfer RNA carries amino acids from cytoplasm to the
    ribosome.
  \item \vocab[]{rRNA} - ribosomal RNA is the major component of the ribosome.
    Catalytic RNAs are also called \vocab[]{ribozymes} and are capable of
    performing specific biochemical reactions, similar to protein enzymes.
    Examples of other noncoding RNA on pg.\ 157.
\end{enumerate}

\subsection{Replication vs. Transcription}
Both replication and transcription involve \vocab[]{template-driven
polymerization} $ \implies $ RNA transcript is also complementary to the DNA template it
was derived from as in replication. Also, transcription goes from $ 5^{\prime} $
to $ 3^{\prime} $ direction and does not require a primer as in replication. It
also has no exonuclease activity i.e. it cannot fix errors it makes. The
sequence of nucleotides that beings transcription is called the
\vocab[]{promoter} and the point where RNA polymerization actually starts is
called the \vocab[]{start site}.

\subsection{Reference Points in Transcription}


\end{document}
